parcel_test.go

```diff
@@ -31,17 +32,28 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
+	res, err := store.Add(parcel)
+	require.Nil(t, err, "error not nil")
```

Comment:
Есть специальная функция для проверки ошибки
```suggestion
	require.NoError(t, err, "error not nil")
```